aten/src/ATen/native/quantized/affine_quantizer.h

```diff
@@ -7,18 +7,39 @@
 namespace at {
 namespace native {
 
+Tensor& quantize_scalar_per_tensor_affine(
+    const float val,
```

Comment:
why is this a float? should we be using Scalar to support different types of Scalars?